app/views/user/check_email_confirmation.html.slim

```diff
@@ -13,7 +13,24 @@
 
 .govuk-grid-row
   .govuk-grid-column-two-thirds-from-desktop
-    = govuk_details(summary_text: "Need us to resend the email?") do
-      = govuk_link_to "I haven't received the email", new_confirmation_path(@user)
+    = govuk_details(summary_text: "Problems with the activation email?") do
+      p
+        strong I haven’t received the email
+      p 
+        | Please wait a few minutes for the email to arrive. If it doesn’t 
+          appear in your inbox, please check your spam or junk email folder. lf you still cannot see the email, we can send it again. 
+      = govuk_link_to "Send me another email", new_confirmation_path(@user)
+
+      br/
       br/
-      = govuk_link_to "I don't have access to the email address I entered", "#"
+      p 
+        strong I don’t have access to the email address 
+      p I used to register You can set up your account using a different email address.
```

Comment:
This looks like you've put in a line break in the wrong place. Should it say "I don't have access to the email address I used to register"?